src/test/java/de/siegmar/logbackgelf/GelfLayoutTest.java

```diff
@@ -181,6 +181,35 @@ public void complex() throws IOException {
         assertNull(jsonNode.get("_exception"));
     }
 
+    @Test
+    public void complexStaticFieldArray() throws IOException {
+        layout.setIncludeRawMessage(true);
+        layout.setIncludeLevelName(true);
+        layout.addStaticField("foo:[bar,barr]");
+        layout.setIncludeCallerData(true);
+        layout.setIncludeRootCauseData(true);
+        layout.start();
+
+        final LoggerContext lc = (LoggerContext) LoggerFactory.getILoggerFactory();
+        final Logger logger = lc.getLogger(LOGGER_NAME);
+
+        final LoggingEvent event = simpleLoggingEvent(logger, null);
+
+        event.setMDCPropertyMap(ImmutableMap.of("mdc_key", "mdc_value"));
```

Comment:
Please remove irrelevant data (here and asserts)

Reply:
Now the test is more simple and objective